frontend/src/components/story/StoryCard.tsx

```diff
@@ -30,8 +32,12 @@ const StoryCard = ({
     return originalYoutubeLink.replace("watch?v=", "embed/");
   };
 
+  // for on-click. StoryTranslationId may be undefined?
```

Comment:
nit: remove console logs

Reply:
I needed it to pass no-unused-vars error